assets/js/leaflet.js

```diff
@@ -90,7 +93,7 @@ class BoatView {
     this.boatId = boatId;
     this.map = map;
     this.trackCoordinates = trackCoordinates;
-    this.marker = Leaflet.marker([0, 0]).addTo(map);
+    this.marker = Leaflet.marker([0, 0], { icon: boatIcon }).addTo(map);
```

Comment:
```suggestion
    this.marker = Leaflet.marker([0, 0], { icon: boatIcon, color: trackColor }).addTo(map);
```

Does this work for coloring?

Reply:
Ah nevermind, I was looking at the customIconMarker, and it uses the color prop in a custom way :(
I wonder if you could get that passed down to the SVG somehow

Reply:
one way would be to use a data url and to interpolate it directly to the SVG string (encoding it as base64 afterwards), but that's kinda ugly.